internal/resources/grafana/resource_dashboard_permission.go

```diff
@@ -97,19 +98,19 @@ Manages the entire set of permissions for a dashboard. Permissions that aren't s
 }
 
 func UpdateDashboardPermissions(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	client, orgID := OAPIClientFromNewOrgResource(meta, d)
+	client, orgID, _ := ClientFromExistingOrgResource(meta, d.Id())
```

Comment:
This client will be deprecated soon. We shouldn't revert from the new OpenAPI-generated client

Reply:
Thanks, I'm looking through the OpenAPI client to understand how to add the endpoints I need for these changes to the client.